inst/workflow/2_metrics/cou0001.yaml

```diff
@@ -62,4 +62,12 @@ steps:
     params:
       dfFlagged: Analysis_Flagged
       nMinDenominator: nMinDenominator
-
+  - name: list
+    output: cou0001
+    params:
+      id: ID
+      input: Analysis_Input
```

Comment:
@jwildfire was it intentional to change all of the `Analysis_*` outputs to remove that prefix and make them lowercase? It seems like we are keeping `Raw_*` and `Reporting_*`, so I wasn't sure why we would want to remove the `Analysis_` prefix for those data objects.

Reply:
okay, i'll keep it as is, but @samussiah - let me know if you have thoughts/preferences and we can log a quick issue to change the naming